test/notebooks/test_notebooks.py

```diff
@@ -175,6 +173,17 @@ def test_fl_with_websockets_and_averaging(
         sy.VirtualWorker(id=n, hook=hook, is_client_worker=False)
 
 
-def test_not_tested_notebooks():
-    """This test must always be last"""
-    assert len(not_excluded_notebooks) == 0, not_excluded_notebooks
+### These tests must always be last
+def test_all_non_excluded_notebooks():
+    untested_notebooks = set(all_notebooks) - set(excluded_notebooks) - set(tested_notebooks)
+    assert len(untested_notebooks) == 0, untested_notebooks
+
+
+def test_all_notebooks_except_translations():
```

Comment:
Yeah, this test is partially redundant, but on purpose. What I'm about to do is set up Github Actions so that the basic notebook tests get run on every PR, but the full notebook tests including the translations only get run when something in the examples directory changes. One or the other of these tests will be excluded from the command line in each check suite action.